doc/en/components/general-getting-started.md

```diff
@@ -114,17 +114,18 @@ import '@webcomponents/custom-elements/custom-elements.min';
 import '@webcomponents/custom-elements/src/native-shim.js';
 ```
 
-## Step 3 - Install Ignite UI for Web Components
+## Step 3 - Install Ignite UI for Web Components and lit-html
 
 1 - Install the Ignite UI for Web Component using **npm**. In this example, we will install the Map web component:
 
 <pre style="background:#141414;color:white;display:inline-block;padding:16x;margin-top:10px;font-family:'Consolas';border-radius:5px;width:100%">
 npm install --save {PackageCore}
 npm install --save {PackageCharts}
 npm install --save {PackageMaps}
+npm install lit-html
```

Comment:
suggestion, all this can be a one liner:
`npm install {PackageCore} {PackageCharts} {PackageMaps} lit-html`